ports/llvm/portfile.cmake

```
@@ -236,7 +272,9 @@ foreach(tool_file IN LISTS LLVM_TOOL_FILES)
     endif()
 endforeach()
 
-vcpkg_copy_tools(TOOL_NAMES ${LLVM_TOOLS} AUTO_CLEAN)
+if(LLVM_TOOLS)
```

This is not a fix for #16222. This is just silencing the error. I believe the reason this is failing is that the preceding glob is scanning for tools in bin, but the tools directory was overridden with -DLLVM_TOOLS_INSTALL_DIR=tools/llvm in #13998.

---

@invertego For your command the folder ${CURRENT_PACKAGES_DIR}/bin/ should be empty. LLVM_TOOLS should be empty too. I cannot reproduce your error with my fix.

---

I checked out the branch for this PR and tried again, and the error did not reproduce, so I was indeed mistaken. Sorry for the confusion.